src/tigerbeetle.zig

```diff
@@ -19,8 +19,9 @@ pub const Account = extern struct {
     timestamp: u64 = 0,
 
     comptime {
+        assert(@alignOf(Account) == 16);
         assert(@sizeOf(Account) == 128);
-        assert(@bitSizeOf(Account) == @sizeOf(Account) * 8);
+        assert(@alignOf(Transfer) == 16);
```

Comment:
Did you mean to change this? The `bitSizeOf` assert says "we expect that there is no implicit struct padding".

Reply:
Nope, just was not sure whether I should assert alignment first or last, and accidetly messed up the whole thing in two different ways while trying to decide :sweat_smile: